tools/depth-quality/depth-quality-model.cpp

```diff
@@ -47,20 +47,26 @@ namespace rs2
 
             // Adjust settings according to USB type
             bool usb3_device = true;
```

Comment:
The exception use case is usb2.
Why not querying if it's USB2 and do
`            int requested_fps = (usb2_device)  ? 15 : 30;`
As I understand it, DDS/GMSL/USB3 should set 30 right?

Reply:
"As I understand it, DDS/GMSL/USB3 should set 30 right?" this is the current code meaning - I assumed it is right

Reply:
The current code works, but I think the logic is more complicated than needed.
You want to check USB2 vs all other
15 vs 30
So better to condition USB2 against all other right?
Currently you have
usb_device
usb3_device..